src/Components/Auth/auth.js

```diff
@@ -10,21 +15,44 @@ import { useState } from "react";
 
 
 export const AuthCheck = () => {
+
+  const {login,setlogin} = useDataContext();
+
+    
+  
     const [username,setusername]=useState('')
       const [pass,setpass]=useState('')
       const [err,seterr]=useState(false)
```

Comment:
instead of using useState you should use useContext
BTW whenever there is a need of useState(more thn 2 or 3) always use useContext